text_scrubber/text_scrubber.py

```diff
@@ -187,6 +231,17 @@ def lowercase(self, on_tokens: bool = False, name: str = 'lowercase') -> 'TextSc
         """
         return self._add_step(name, str.lower, on_tokens)
 
+    def normalize_unicode(self, form: str, on_tokens: bool = False, name: str = 'normalize_unicode') -> 'TextScrubber':
```

Comment:
If there's a limited number of valid values, consider using `Literal["NFC", "NFKC", "NFD", "NFKD"]` as type